EapWebInterface/src/main/java/de/hpi/unicorn/application/pages/input/generator/GeneratePanel.java

```diff
@@ -98,34 +126,58 @@ public void onSubmit() {
         addTimestampField();
         addEventTypeDropDown();
         addSubmitButton();
-   }
+        addExportValuesButton();
+        addImportField();
+        addImportSubmitButton();
+    }
 
     /**
      * Add field to specify the number of events to generate.
      */
     private void addEventCountField() {
-        final TextField<Integer> eventCountField = new TextField<>("eventCountField", new PropertyModel<Integer>(this, "eventCount"));
+        eventCountField = new TextField<>("eventCountField", new PropertyModel<Integer>(this, "eventCount"));
         eventCountField.setRequired(true);
         eventCountField.add(new RangeValidator<Integer>(1, MAXIMUM_EVENTCOUNT));
+        eventCountField.add(new AjaxFormComponentUpdatingBehavior("change") {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "change" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AS1192)